docs/src/codegen.md

```diff
@@ -156,55 +244,123 @@ playwright codegen --color-scheme=dark playwright.dev
+######
+* langs: js
 
-### Preserve authenticated state
+<img width="1394" alt="Codegen generating code for tests for bing maps showing timezone, geolocation as Rome, Italy and in Italian language" src="https://user-images.githubusercontent.com/13063165/220931996-d3144421-8d3b-4f9f-896c-769c01566c01.png" />
+
+######
+* langs: java
+
+<img width="1394" alt="Codegen generating code for tests for bing maps showing timezone, geolocation as Rome, Italy and in Italian language java" src="https://user-images.githubusercontent.com/13063165/220932268-9012163f-7673-4072-aa91-13b3c8f57799.png" />
 
-Run `codegen` with `--save-storage` to save [cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies) and [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) at the end of the session. This is useful to separately record an authentication step and reuse it later in the tests.
+######
+* langs: python
 
-After performing authentication and closing the browser, `auth.json` will contain the storage state.
+<img width="1394" alt="Codegen generating code for tests for bing maps showing timezone, geolocation as Rome, Italy and in Italian language python" src="https://user-images.githubusercontent.com/13063165/220932413-f2943956-dd38-4560-94b9-51968076210d.png" />
+
+
+######
+* langs: csharp
+
+<img width="1394" alt="Codegen generating code for tests for bing maps showing timezone, geolocation as Rome, Italy and in Italian language csharp" src="https://user-images.githubusercontent.com/13063165/220932688-a47df2a8-332b-47a4-9580-7d351def9f50.png" />
+
+### Preserve authenticated state
+
+Run `codegen` with `--save-storage` to save [cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies) and [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) at the end of the session. This is useful to separately record an authentication step and reuse it later when recording more tests. 
 
 ```bash js
-npx playwright codegen --save-storage=auth.json
+npx playwright codegen github.com/microsoft/playwright --save-storage=auth.json
 ```
 
 ```bash java
-mvn exec:java -e -D exec.mainClass=com.microsoft.playwright.CLI -D exec.args="codegen  --save-storage=auth.json"
+mvn exec:java -e -D exec.mainClass=com.microsoft.playwright.CLI -D exec.args="codegen github.com/microsoft/playwright  --save-storage=auth.json"
 ```
 
 ```bash python
-playwright codegen --save-storage=auth.json
+playwright codegen github.com/microsoft/playwright --save-storage=auth.json
 ```
 
 ```bash csharp
-pwsh bin/Debug/netX/playwright.ps1 codegen --save-storage=auth.json
+pwsh bin/Debug/netX/playwright.ps1 codegen github.com/microsoft/playwright --save-storage=auth.json
 ```
 
-<img width="1264" alt="Screenshot 2022-08-03 at 13 28 02" src="https://user-images.githubusercontent.com/13063165/182599605-df2fbd05-622b-4cd7-8a32-0abdfea7d38d.png" />
+######
```

Comment:
I'd not add the following 4 screenshots, I don't see much value in them, since its just a goto.

Reply:
in js yes but in other langs it actually shows you the auth.json and is much more interesting to see it also helps you to see what you should get if you run the command cause i wasnt sure if i was doing it right cause it doesnt show the saving of auth so I do find it useful as a this is how it should look like kinda thing